lib/ymodel/lib/ymodel/loadable.rb

```diff
@@ -10,6 +10,7 @@ module Loadable
     # overwrite the default ymodel filename associated with that model.
     def source_file(filename)
       @source = filename
+      @records = nil
```

Comment:
Because I wanted the fixtures used as source files for YModel classes like OutputElements. These classes were already initialised/triggered, so I retriggered them with the fixture files.
Maybe you know a better way to point the YModel classes to the fixtures folder?

Reply:
I'm not to sure why you would want to add fixture files. Could a mock isolate the test you are trying to write?